`./gradlew jettyRunForeground` don't work

Reopened because of broken link, see: https://github.com/mapfish/mapfish-print/issues/295#issuecomment-136284138 and next

---

specified in https://github.com/mapfish/mapfish-print/tree/development/examples
=> 
```
Picked up _JAVA_OPTIONS:  -Dorg.eclipse.swt.browser.XULRunnerPath=~/xulrunner -XX:MaxPermSize=256m  -Dorg.eclipse.swt.browser.XULRunnerPath=~/xulrunner -XX:MaxPermSize=256m 
Defining custom ‘build’ task is deprecated when using standard lifecycle plugin has been deprecated and is scheduled to be removed in Gradle 3.0

FAILURE: Build failed with an exception.

* What went wrong:
Task 'jettyRunForeground' not found in root project 'mapfish-print3'.

* Try:
Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 24.465 secs
```

--- a/examples/Readme.rst
+++ b/examples/Readme.rst
@@ -3,49 +3,34 @@ Summary
 This repository is a submodule of mapfish-print and contains e2e tests for the pdf and image generation.  It contains the code to
 start a Geoserver instance and run junit integration tests against the server to test as many of the mapfish-print options as possible.
 
-Due to the difficulty of verification there are two modes of execute.
+To run the integration tests::
 
-An interactive mode and a automated mode.  The interactive mode contains steps where a developer has to validate the responses.
+    ./gradlew examples:test
 
-To use module::
-
-    git clone --recurse-submodules https://github.com/mapfish/mapfish-print.git
-    cd mapfish-print
-    ./gradlew interactiveTest # for interactive and automated tests
-    ./gradlew test # for automated tests only
-    
-    
-Both tasks are gradle test tasks and more details on how to run single tests or subgroups of tests can be understood by referring to:
+The task is a gradle test task and more details on how to run single tests or subgroups of tests can be understood by referring to:
 
     http://www.gradle.org/docs/current/userguide/java_plugin.html#sec:java_test
 
 
-Test Server and SamplePages
----------------------------
-The geoserver in this project is configured with several webpages that show maps requesting tiles in the various layer types
-(TMS, WMS, WMTS, Vector, OSM, etc...)  These pages can be used to see how openlayer interacts with these different layer types.  The
-pages also have print buttons for submitting a print request for the given map.
+Test client
+------------
 
-To experiment with these pages:
+The test server includes a client which can be used for testing. To start the server, run::
 
-1. Open a terminal
-2. `./gradlew jettyRunForeground`
-3. Open a web browser
-4. Enter `http://localhost:9876/e2egeoserver` in the location input
+     ./gradlew examples:jettyRun
 
-The main page contains a list of all the pages and provides links to the respective pages.  Each page has a short description
-explaining what should be visible and printed.
+
+This will start a GeoServer at localhost:9876 but also a MapFish Print with a simple client that can accessed
+at http://localhost:8080/print-servlet/
+
 
 Writing Tests
 -------------
 
-There are two types of tests.  All classes that end in _InteractiveTest_ will be ran only during the testInteractive task.  All other junit
-tests will be ran during the normal test task.
-
 By default the test server is in daemon mode, which mean that the servers will be run in a background thread and be shutdown when
-the build completes.  In order to be able to run the tests in a IDE there is a one can run::
+the build completes.  In order to be able to run the tests in a IDE one can run::
 
-     ./gradlew jettyRunForeground
+     ./gradlew examples:jettyRun
 
-This will start the test servers in non-daemon mode allowing one to start the server and then run tests you IDE against that server for
+This will start the test servers in non-daemon mode allowing one to start the server and then run tests in your IDE against that server for
 development.

